### DISCUSSION TITLE
a problem with immich android

### DISCUSSION THREAD

**Discussion by Alihkhawaher** [immich-app/immich#9956] [Q&A]
### The bug

I just installed immich on a fresh installation of ubuntu 24 LTC and used snaps to install immich-distribution, I configured everything and installed immich android app. Everything was amazing except for one thing. The android app did not upload and meta data for all pictures, all photos have the same date, without any gps info or whatso ever. 

To be noted that I installed the server twice without clearing android app data or re-installing

### The OS that Immich Server is running on

Ubuntu 24.04 LTS

### Version of Immich Server

1.105.1

### Version of Immich Mobile App

1.105.0

### Platform with the issue

- [X] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
default 
```

### Your .env content

```Shell
UPLOAD_LOCATION=./library
# The location where your database files are stored
DB_DATA_LOCATION=./postgres

# The Immich version to use. You can pin this to a specific version like "v1.71.0"
IMMICH_VERSION=release

# Connection secret for postgres. You should change it to a random password
DB_PASSWORD=  

# The values below this line do not need to be changed
###################################################################################
DB_USERNAME=postgres
DB_DATABASE_NAME=immich

```

### Reproduction steps

```bash
install server and backup photos, then re-install server with the same android app data
```

**Comment by bo0tzz** [comment]
This seems like a metadata processing issue. Since you're using an installation method that doesn't use our docker images, and thus is somewhat likely to have the wrong image processing dependencies, I don't think we can be of much help.

**Reply by Alihkhawaher** [reply]
Thank you for the reply, can I have some tips on how to check decencies for metadata? Actually I tried running meta data job before creating this issue, it runs for a few minutes on 5000 photos then it stops. I tried extracting the logs but I couldnt find something usefull,

I used the following command

sudo journalctl -u snap.immich* -g error

**Reply by moritzbindewald** [reply]
> Thank you for the reply, can I have some tips on how to check decencies for metadata? Actually I tried running meta data job before creating this issue, it runs for a few minutes on 5000 photos then it stops. I tried extracting the logs but I couldnt find something usefull,
> 
> 
> 
> I used the following command
> 
> 
> 
> sudo journalctl -u snap.immich* -g error
> 
> 

Why don't u just use docker? If the issue persists over there, reach out again?

**Reply by Alihkhawaher** [reply]
I already started with snap and I liked how easy and fast it is to install it one command. It got installed in 2 minutes and ready. I decided to try at least to make it work before giving up.

**Reply by Alihkhawaher** [reply]
Hello,

I reformatted the server and installed it using docker, I even used the apk from github but I still have the same issue.

**Reply by alextran1502** [reply]
snap package is not distributed by us so I suspect there are issues there. I highly suggest follow the recommended installation method so we can help if you run into any issues

**Reply by Alihkhawaher** [reply]
Yes, I followed the installations instructions from here:

https://immich.app/docs/overview/quick-start/

snap was my first try, now I am using the official method but I still have the same issue. 

I did not mention this before but gps location works with only 4 photos. 

In google photos all my photos have location since I like keeping track of photos location

**Reply by bo0tzz** [reply]
Google tracks your location and uses that metadata for photos as well. Immich can only use what your camera app puts in the EXIF data, so please check that to determine whether Immich is missing anything.

**Reply by Alihkhawaher** [reply]
> Google tracks your location and uses that metadata for photos as well. Immich can only use what your camera app puts in the EXIF data, so please check that to determine whether Immich is missing anything.

I am really really surprised, it was this all along. I am 100% sure I enabled location in my camera app. I remember clearly enabling it. Also they appear in the photo map so I assumed it was enabled all along. I am sure it got disabled during one of Samsung updates.

The good news of this is that it was not snaps installation method that is not working.

Thank you for taking time to reply to me
